Lets Do This/Views/Campaign/LDTCampaignDetailCampaignCell.m

```diff
@@ -95,11 +107,35 @@ - (void)setCoverImageURL:(NSURL *)coverImageURL {
     }];
 }
 
-- (void)setDisplaySubmitReportbackButton:(BOOL)displaySubmitReportbackButton {
-    _displaySubmitReportbackButton = displaySubmitReportbackButton;
-    if (displaySubmitReportbackButton) {
-        // @todo: Create public SubmitReportbackButtonTitle property.
-        [self.submitReportbackButton setTitle:@"Prove it".uppercaseString forState:UIControlStateNormal];
+- (void)setDisplayCampaignDetailsView:(BOOL)displayCampaignDetailsView {
+    _displayCampaignDetailsView = displayCampaignDetailsView;
+    if (!_displayCampaignDetailsView) {
+        self.diagonalShapeLayer.hidden = YES;
+        // Hack to avoid UILabels preventing  campaignDetailsView from fully collapsing.
```

Comment:
Attempting to use the `.hidden` property does hide the `UILabels`, but they still take up space:

![simulator screen shot dec 21 2015 12 10 24 pm](https://cloud.githubusercontent.com/assets/1236811/11944047/89b70c5e-a7f7-11e5-8f61-876409cc8911.png)